main.js

```diff
@@ -129,4 +129,30 @@ const checkLowerCase = (input, event) => {
 getForm.addEventListener('submit', (e) => {
   e.preventDefault();
   checkLowerCase(getEmail);
-});
\ No newline at end of file
+});
+
+// Start LocalStorage
+
+const getName = document.querySelector('input[type="text"]');
+const getTextArea = document.querySelector('textarea');
+
+const dataStore = JSON.parse(localStorage.getItem('dataStore')) || [];
+
+const addData = ((name, email, text) => {
+  dataStore.push({ name, email, text });
+  localStorage.setItem('dataStore', JSON.stringify(dataStore));
+  return { name, email, text };
+});
+
+const showData = (({ name, email, text }) => {
+  getName.value = name;
+  getEmail.value = email;
+  getTextArea.value = text;
+});
+
+dataStore.forEach(showData);
+
+getForm.addEventListener('submit', (e) => {
```

Comment:
Kindly make sure the data exist after the page load and create one object for it on the local storage as the previous reviewer requested. Tip: Evenlistener needs to listen to the changes on the input, not after the submit. Check [this](https://huseyinbiyik.github.io/) to get inspired.

Currently your project:
![Screen Recording 2023-01-07 at 4 40 27 PM](https://user-images.githubusercontent.com/25887253/211153958-06da467b-53fa-474b-bf15-833e0e02f9d1.gif)